From 458e8a851608b8308b564e6cbfe95314629dccda Mon Sep 17 00:00:00 2001
From: Carmine Scarpitta <cscarpit@cisco.com>
Date: Sat, 23 Nov 2024 08:46:50 +0100
Subject: [PATCH 5/5] tools: Add missing keyword `node-msd` in frr-reload

Fix the error below:

```
[58564|mgmtd] sending configuration
[58565|zebra] sending configuration,
line 14: % Unknown command[52]:  max-segs-left 3
line 18: % Unknown command[52]:  max-end-pop 3
line 22: % Unknown command[52]:  max-h-encaps 2
line 26: % Unknown command[52]:  max-end-d 5
[58573|isisd] sending configuration
[58565|zebra] Configuration file[/etc/frr/frr.conf] processing failure: 2
```

Signed-off-by: Carmine Scarpitta <cscarpit@cisco.com>
---
 tools/frr-reload.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tools/frr-reload.py b/tools/frr-reload.py
index baa6285f45..33d6cf0444 100755
--- a/tools/frr-reload.py
+++ b/tools/frr-reload.py
@@ -256,7 +256,9 @@ ctx_keywords = {
     "router rip": {},
     "router ripng": {},
     "router isis ": {
-        "segment-routing srv6": {},
+        "segment-routing srv6": {
+            "node-msd": {},
+        },
     },
     "router openfabric ": {},
     "router ospf": {},
-- 
2.39.5

